scripts/end_to_end_test/end_to_end_test.yaml

```diff
@@ -244,7 +244,7 @@ steps:
 
 ---
 test_description: >
-  Test synergetic contract fees
+  DISABLED - Test synergetic contract fees
```

Comment:
How come disabled?

Reply:
Ran out of time, I mentioned I would ask atilla to fix them (rely on wealth creation)